clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -318,7 +318,8 @@ public class ProducerConfig extends AbstractConfig {
 
     /** <code> transaction.timeout.ms </code> */
     public static final String TRANSACTION_TIMEOUT_CONFIG = "transaction.timeout.ms";
-    public static final String TRANSACTION_TIMEOUT_DOC = "The maximum amount of time in ms that the transaction coordinator will wait for a transaction status update from the producer before proactively aborting the ongoing transaction." +
+    public static final String TRANSACTION_TIMEOUT_DOC = "The maximum amount of time in milliseconds that a transaction will remain open before the coordinator proactively aborts it. " +
+            "The start of the transaction is set at the time that the first partition is added to it. " +
             "If this value is larger than the transaction.max.timeout.ms setting in the broker, the request will fail with a <code>InvalidTxnTimeoutException</code> error.";
```

Comment:
nit: could we put `<code>` blocks around `transaction.max.timeout.ms` while we're in here?

Reply:
Done in [a1119c1](https://github.com/apache/kafka/pull/13129/commits/a1119c16f55555d8c3cedf554d62e8112085b2f6)